Contest_API.md

```diff
@@ -1571,65 +1571,74 @@ If the client fails to respond to multiple requests over a period of
 time (configured for each contest), it will be assumed deactivated and
 automatically removed from future callbacks.
 
-The following endpoint is associated with the webhook:
+The following endpoints is associated with the webhook:
 
-| Endpoint   | Mime-type        | Required? | Source @WF | Description                            |
-| ---------- | ---------------- | --------- | ---------- | -------------------------------------- |
-| `/webhook` | application/json | yes       | CCS        | List or register for webhook callbacks |
+| Endpoint         | Mime-type        | Required? | Source @WF | Description                                                                                                          |
+| ---------------- | ---------------- | --------- | ---------- | -------------------------------------------------------------------------------------------------------------------- |
+| `/webhooks`      | application/json | yes       | CCS        | JSON array of all webhook callbacks with elements as defined in the table below. Also used to register new webhooks. |
+| `/webhooks/<id>` | application/json | yes       | CCS        | JSON object of a single webhook callback with elements as defined in the table below                                 |
```

Comment:
We could have `/contests/<id>/webhooks` (and `/contests/<id>/webhooks/<id>`) instead (or in addition).

Putting it under contests allows for saner defaults. Not putting it at the top level makes it harder to follow *all* contests, which might be useful at an onsite event, but is not very good at all in an online setting.

Reply:
Decided it should at least be at the global level `/contests/webhooks`.